lib/queryBuilder/graphUpserter/UpsertNode.js

```diff
@@ -154,6 +128,12 @@ function hasOption(node, optName) {
   }
 }
 
+function decideType(node, defaultType, option, optionType) {
+  return hasOption(node, option)
+    ? optionType || UpsertNodeType.None
+    : defaultType;
+}
+
```

Comment:
@koskimas optionType is always null currently, but I was going to use this to implement the `update: true` option also, where it will have another value.